docs-2.0/5.configurations-and-logs/1.configurations/3.graph-config.md

```diff
@@ -191,4 +191,13 @@ For more information about audit log, see [Audit log](../2.log-management/audit-
 |`enable_http2_routing`  |`false`   |Whether to enable HTTP2 for RPC communications. Enabling it will slightly affect performance.|Yes|
 |`stream_timeout_ms`    |`30000`  | The timeout for the HTTP stream. Unit: ms.|Yes|
 
+
+## Zone Configuration
+
+| Name                              | Default Value | Description                                                                                                                             | Runtime Dynamic Modification Supported |
+| :-------------------------------- | :------------ | :-------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------- |
+| `assigned_zone`                   | Empty         | When the Zone feature is enabled, set an existing Zone name for directed access. See [Managing Zones](../../4.deployment-and-installation/5.zone.md) for details. | No                                    |
+| `prioritize_intra_zone_reading`   | `false`       | When set to `true` and a Zone is specified in `assigned_zone`, it will prioritize reading replica data within that Zone. If reading fails in that Zone, it will decide based on `stick_to_intra_zone_on_failure` whether to read data from other Zones.<br/>When set to `false`, it will read from the leader partition replicas. | No                                    |
```

Comment:
We do not retry. If reading fails in that zone, then either we return failure or return partial success. A more accurate description is "if no replica is found in the same zone ..."